desktop/core/src/desktop/lib/thrift_util.py

```diff
@@ -557,6 +557,14 @@ def wrapper(*args, **kwargs):
               raise
         except Exception as e:
           logging.exception("Thrift saw exception (this may be expected).")
+          if "Read timed out." in str(e):
```

Comment:
Is this string specific to impala? If so, perhaps see if we can include other engines as well?

Reply:
We are getting a read timeout from Impala. This string is actually coming from our underlying REST client library, so it should be engine agnostic.

Reply:
Excellent.